addon/components/hyper-table-v2/index.ts

```diff
@@ -40,6 +42,8 @@ export default class HyperTableV2 extends Component<HyperTableV2Args> {
       args.handler.fetchRows();
       this.computeScrollableTable();
     });
+
+    this.hypertableInstanceID = window.crypto.randomUUID();
```

Comment:
In hypertableV2, you are using `window.crypto` and in hypertableV1, you are using `crypto` (without window), is it normal?